File: Lógica/Lista2/ExPySala/ex3.py
valor1 = float(input("Digite o primeiro valor: "))
valor2 = float(input("Digite o segundo valor: "))

if valor1 > valor2:
    d = valor1 - valor2
    print(f"Diferença entre {valor1} e {valor2}: {d}")
else:
    d = valor2 - valor1
    print(f"Diferença entre {valor2} e {valor1}: {d}")

File: Lógica/Lista2/ExPySalaC/ex3.c
#include <stdio.h>

void main()
{   
    float v1,v2,d;
    
    printf("Digite o primeiro valor: ");
    scanf("%f",&v1);
    
    printf("Digite o segundo valor: ");
    scanf("%f",&v2);
    
    if (v1 >= v2){
        d = v1 - v2;
        printf("Diferença entre %f e %f: %f",v1,v2,d);
    }
    else{
        d = v2 - v1;
        printf("Diferença entre %f e %f: %f",v2,v1,d);
    }
}